src/constants.js

```diff
@@ -1,5 +1,18 @@
 'use strict'
 
 module.exports = {
-  COOKIE_KEY: 'DefraSession'
+  PAGE_TITLE_ERROR_PREFIX: 'Problem: ',
+  SERVICE_NAME: 'Waste Permits',
+  GDS_NAME: 'GOV.UK',
+
+  COOKIE_KEY: 'DefraSession',
+
+  Routes: {
```

Comment:
👍 having a central place for managing and listing the paths in the app. Such a good idea, it got me thinking should we do the same for the page heading as well e.g.

```javascript
Routes: {
  ROOT: {
    path: '/',
    pageHeading: 'Waste Permits Home Page'
  },
  ERROR: {
    path: '/error',
    pageHeading: 'Something went wrong'
  }
}
```

I think there is a benefit to seeing the titles and paths together in one place but interested in what you think.

Reply:
Yes I agree, I was considering whether or not to before, I'm happy to make that change.